src/main/java/frc/robot/subsystems/Drivetrain.java

```diff
@@ -89,13 +97,19 @@ public class Drivetrain extends SubsystemBase {
   // modules needed to distinguish in chooser
   private Module m_prevModule;
 
+  GameControllerDriverConfig m_config = new GameControllerDriverConfig(null, m_drivetrainTab, m_hasResetYaw);
+
+  GameController m_Controller = new GameController(OIConstants.kDriverJoy);
+
   /**
    * Creates a new Swerve Style Drivetrain.
    * @param drivetrainTab the shuffleboard tab to display drivetrain data on
    * @param swerveModulesTab the shuffleboard tab to display module data on
    */
   public Drivetrain(ShuffleboardTab drivetrainTab, ShuffleboardTab swerveModulesTab) {
 
+    m_config.configureControls();
+
```

Comment:
don't make a new driver controller or game controller in drivetrain, you don't even use it for anything